lib/routes/user/home/account_page.dart

```diff
@@ -19,7 +18,7 @@ import 'package:breez/logger.dart';
 
 const DASHBOARD_MAX_HEIGHT = 188.0;
 const DASHBOARD_MIN_HEIGHT = 70.0;
-const FILTER_MAX_SIZE = 70.0;
+var FILTER_MAX_SIZE = 70.0;
```

Comment:
Don't change this constant, if you need add another constant and use calculated getter to get the result you need based on the current context.